modules/installation-requirements-user-infra.adoc

```diff
@@ -197,6 +197,88 @@ On your z/VM instances, set up:
 * 16 GB for the temporary {product-title} bootstrap machine
 endif::ibm-z[]
 
+ifdef::ibm-power[]
+
+[id="minimum-ibm-power-system-requirements_{context}"]
+== Minimum IBM Power Systems requirements
```

Comment:
I didn't realize until after reading through this that there was a recommended requirements section too. I feel like people might come to this section first and then stop before noticing that there is a different section which is actually what we recommend.

I'd consider putting the recommended section first, before the minimum requirement section, for this reason.

Additionally, you should add an introductory sentence both the minimum and recommended sections, just to reiterate that one is the bare minimum requirements, and that the other is what is actually recommended (for better performance, or whatever the reason is).

Reply:
As discussed we will leave this to be consistent with the section in the IBM Z docs.